### PR TITLE
Support PHPUnit versions 5, 6 and 7

### PR DESCRIPTION
**Pull request by carusogabriel**
Reopening #124 

/cc @alcaeus

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
         "doctrine/cache":          "^1.4.2"
     },
     "require-dev": {
-        "phpunit/phpunit":                       "~4",
+        "phpunit/phpunit":                       "~4.8.36|~5.6|~6.5|~7.0",
```
Comment (fabpot) [inline composer.json]: should be `^4.8.36`

Comment (Majkl578) [reply composer.json]: Isn't that irrelevant? There won't ever be 4.9.

**Comment by alcaeus** [comment]
@carusogabriel can you rebase and fix the conflict? Thanks!

**Comment by carusogabriel** [comment]
@alcaeus Done :smile:

**Comment by alcaeus** [comment]
Thanks @carusogabriel!

**Pull request merged**